app/config/collections.php

```diff
@@ -2327,6 +2327,17 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('buildTime'),
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
```

Comment:
Why did we add this? This should be part of the build itself 👍🏻

Reply:
fixed